provider-blackduck/src/main/java/com/synopsys/integration/alert/provider/blackduck/BlackDuckProperties.java

```diff
@@ -147,7 +149,7 @@ public BlackDuckServerConfig createBlackDuckServerConfig(IntLogger logger, int b
     }
 
     public BlackDuckServerConfigBuilder createServerConfigBuilderWithoutAuthentication(IntLogger logger, int blackDuckTimeout) {
-        BlackDuckServerConfigBuilder blackDuckServerConfigBuilder = new BlackDuckServerConfigBuilder();
+        BlackDuckServerConfigBuilder blackDuckServerConfigBuilder = new BlackDuckServerConfigBuilder(KEYS.apiToken);
```

Comment:
With the latest BDC, the default BlackDuckServerConfigBuilder() does not include the keys for API or User/Pass. We now need to specify we want the KEYS which include the ApiToken